service/awsconfig/framework.go

```diff
@@ -275,9 +275,6 @@ func newResourceRouter(config FrameworkConfig) (*framework.ResourceRouter, error
 			Logger:             config.Logger,
 			RandomkeysSearcher: keyWatcher,
 
-			HandledVersionBundles: []string{
-				"2.1.0",
-			},
```

Comment:
I am not doing it for earlier versions now because there are all multiple version bundles per resource version and this is just odd. This here should be good enough as example for future iterations and all the legacy crap is going to go away eventually.

Reply:
Agree only makes sense to sort out the latest version bundles.